architecture/authentication.adoc

```diff
@@ -0,0 +1,268 @@
+= Authentication
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title: 
+
+toc::[]
+
+== Overview
+
+The authentication layer is responsible for identifying the user associated with requests to the OpenShift API. Information about the requesting user is then used by the authorization layer to determine whether the request should be allowed.
+
+== API Authentication
+
+Requests to the OpenShift API are authenticated using the following methods:
+
+* OAuth Access Token
+** Obtained from the OpenShift OAuth server using the `/oauth/authorize` and `/oauth/token` endpoints
```

Comment:
I mentioned that later in this doc, and in the using auth doc... since that simply redirects to /oauth/authorize, was trying to simplify the description of the path all OAuth token requests funnel to. If you don't think if confuses the issue, I can add it here as well